ci/setup_grpc.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+export DEBIAN_FRONTEND=noninteractive
+
+if ! type cmake > /dev/null; then
+    #cmake not installed, exiting
+    exit 1
+fi
+export BUILD_DIR=/tmp/
+export INSTALL_DIR=/usr/local/
+pushd $BUILD_DIR
+git clone --recurse-submodules -b v1.34.0 https://github.com/grpc/grpc
+cd grpc
+mkdir -p cmake/build
+pushd cmake/build
+cmake -DgRPC_INSTALL=ON \
+    -DgRPC_BUILD_TESTS=OFF \
+    -DCMAKE_INSTALL_PREFIX=$INSTALL_DIR \
+    ../..
+make -j 6
```

Comment:
Is this `-j 6` necessary to be specified manually?

Reply:
This seems to be optimal value as I tested. `make -j` creates lots of jobs simultaneously and the process crashes in end. `make` is too slow.

Reply:
Usually this works well:
```
make -j $(nproc)
```
`nproc` returns the number of cores a machine has (or a docker container has assigned to it).

Reply:
thanks, changed to `nproc` :)